app/api/routes.py

```diff
@@ -17,12 +17,14 @@ def resources():
     return get_resources()
 
 
-@bp.route('/resources/<int:id>', methods=['GET', 'PUT'])
+@bp.route('/resources/<int:id>', methods=['GET', 'PUT', 'POST'])
```

Comment:
The route needs to be `POST /resources` due to the fact that there is no ID for a brand new resource until the database assigns one.

Please move the `POST` method and the call to `create_resource` up to this method here https://github.com/OperationCode/resources_api/blob/63c67b898caf797eb8497c332a37a187cb05f2e8/app/api/routes.py#L15